opam_win_repo/repo

```diff
@@ -0,0 +1 @@
+opam-version: "2.0"
```

Comment:
So the custom win repo has to stick with the former 2.0 version?
Or is it a typo ?

I noticed it's also the case in conf-lib-curl, conf-mingw and ocurl. 
Will that mean that we will have two opam versions in parallel, 2.2 for most of the stuff, and 2.0 for the rest?

Reply:
It's a different version number: https://opam.ocaml.org/doc/Manual.html#repofield-opam-version.